### ISSUE TITLE
Freezing when clicking the dat:// link above Library items

### ISSUE THREAD

**Issue by m455** [beakerbrowser/beaker#1364]
Operating system: Windows 10

This is:

- [x] a bug
- [ ] general feedback

What happens:
When I click the link, as circled in red in the image below, my beaker browser freezes, becomes unresponsive, and then I have to kill the process. Upon restarting Beaker Browser, it realizes that it has "crashed" and asks if I would like to restart the old session, or start new.

All items in my _Library_ are in tact, so no data loss is present in this bug.

![bug](https://user-images.githubusercontent.com/25066464/55043232-aa16e400-500b-11e9-9602-881f8718e999.png)

Extra comment:
Beaker Browser is so fun, and I'm so happy you people are putting in your time and effort into this.

~m455

**Comment by tiotrom** [comment]
I think many of us experience this with the latest beaker. Crashes on me a lot too. Cannot use it. I think others reported as well and they are aware and trying to fix.

**Comment by pfrazee** [comment]
Yeah, this is a known issue. We're waiting on Electron to merge and release this fix: https://github.com/electron/electron/pull/17267

**Comment by pfrazee** [comment]
Closing as a duplicate of https://github.com/beakerbrowser/beaker/issues/1333. So sorry about the issue.

**Comment by m455** [comment]
Gotcha! Thanks for this info you two!

Yeah there are a few places where I know to avoid clicking to prevent
crashes.

**Comment by pfrazee** [comment]
Lol yeah, I've developed the same sixth sense.
